gatsby-config.js

```diff
@@ -536,6 +536,7 @@ module.exports = {
             process.env.FEEDBACK_RECAPTCHA_TOKEN ||
             '6Lfn8wUiAAAAANBY-ZtKg4V9b4rdGZtJuAng62jo',
         },
+        trailingSlash: 'always',
```

Comment:
this has to go in top level exports, not in `newrelic`

https://www.gatsbyjs.com/docs/reference/config-files/gatsby-config/#trailingslash